web-ng/src/app/components/layer-dialog/layer-dialog.component.css

```diff
@@ -152,3 +151,11 @@
 .trash-icon {
   color: #707074;
 }
+
+.drag-icon {
+  display: block;
+  margin: 0px auto;
+  cursor: move;
+  padding: 5px;
+  color: #dadde1;
```

Comment:
Could we turn this into scss and use `mat-color($foreground, lighter)` instad?

I was trying to follow https://material.angular.io/guide/theming-your-components but was unsuccessful.. Any idea how to set that up?